vms/job/templates/job/list_sign_up.html

```diff
@@ -4,6 +4,149 @@
+                <div class="form-group has-error">
+                    <label class="col-md-2 control-label">{% trans "State/Province" %}</label>
+                    <div class="col-md-8">
+                        <input class="form-control" placeholder="{% blocktrans %}Search by State/Province{% endblocktrans %}" type="text" name="state" id="job_state" value="{% if form.state.value %}{{ form.state.value }}{% endif %}">
+                        <p class="help-block">
+                            <strong>
+                                        {% for error in form.state.errors %}
+                                            {{ error }}
+                                        {% endfor %}
+                                    </strong>
+                        </p>
+                    </div>
+                </div>
+                {% else %}
+                <div class="form-group">
+                    <label class="col-md-2 control-label">{% trans "State/Province" %}</label>
+                    <div class="col-md-8">
+                        <input class="form-control" placeholder="{% blocktrans %}Search by State/Province{% endblocktrans %}" type="text" name="state" id="job_state" value="{% if form.state.value %}{{ form.state.value }}{% endif %}">
+                    </div>
+                </div>
+                {% endif %}
+                {% if form.country.errors %}
+                <div class="form-group has-error">
+                    <label class="col-md-2 control-label">{% trans "Country" %}</label>
+                    <div class="col-md-8">
+                        <input class="form-control" placeholder="{% blocktrans %}Search by Country{% endblocktrans %}" type="text" name="country" id="job_country" value="{% if form.country.value %}{{ form.country.value }}{% endif %}">
+                        <p class="help-block">
+                            <strong>
+                                        {% for error in form.country.errors %}
+                                            {{ error }}
+                                        {% endfor %}
+                                    </strong>
+                        </p>
+                    </div>
+                </div>
+                {% else %}
+                <div class="form-group">
+                    <label class="col-md-2 control-label">{% trans "Country" %}</label>
+                    <div class="col-md-8">
+                        <input class="form-control" placeholder="{% blocktrans %}Search by Country{% endblocktrans %}" type="text" name="country" id="job_country" value="{% if form.country.value %}{{ form.country.value }}{% endif %}">
+                    </div>
+                </div>
+                {% endif %}
+        </fieldset>
+        <div class="form-group">
+            <div class="col-md-12 col-md-offset-2">
+                <button class="btn btn-primary" id="submit_job" type="submit">{% trans "Search" %}</button>
```

Comment:
Ideally, all submit buttons in VMS has the same ID so they have a common function to click this submit button in automated tests. I think we shouldn't break that if possible.